src/types/index.ts

```diff
@@ -1,4 +1,13 @@
 /**
  * Type describing available languages
  */
-export type Language = "fi" | "en";
\ No newline at end of file
+export type Language = "fi" | "en";
+
+/**
+ * NavButtonProps
+ */
+export interface NavButtonProps {
```

Comment:
This Prop interface could now be moved into the NavButton component and renamed Props with the usual "Component Properties" description